### ISSUE TITLE
Collaboration

### ISSUE THREAD

**Issue by jeremylcarter** [x2bool/duckdb-net#1]
Hi @x2bool 

@Giorgi has been working hard on DuckDB.NET available here https://github.com/Giorgi/DuckDB.NET

I notice that you have also been working on .NET bindings / ADO support for DuckDB. 

I think we should all collaborate on one project.

**Comment by x2bool** [comment]
Yeah. We started in parallel. But now I think we should focus on the library you've mentioned. I will probably archive this project, if @Giorgi is going to continue developing his project.

**Comment by Giorgi** [comment]
Yes, I plan to continue implementing the missing featues. @x2bool @jeremylcarter Are you interested in making contributions?
